example/quantization/imagenet_inference.py

```diff
@@ -99,7 +100,34 @@ def score(sym, arg_params, aux_params, data, devs, label_name, max_num_examples,
             logger.info(m.get())
 
 
-def benchmark_score(symbol_file, ctx, batch_size, num_batches, data_layer_type, logger=None):
+def low_precison_convert(model_name, low_precision, sym, arg_params, aux_params, excluded_sym_names=[]):
+    if low_precision == 'bfloat16':
+        if model_name.find('imagenet1k-resnet-152') != -1:
+            excluded_sym_names += ['conv0']
+        elif model_name.find('imagenet1k-inception-bn') != -1:
+            excluded_sym_names += ['conv_1']
+        elif model_name.find('resnet') != -1 and model_name.find('v1') != -1:
+            excluded_sym_names += ['resnetv10_conv0_fwd']
+        elif model_name.find('resnet') != -1 and model_name.find('v2') != -1:
+            excluded_sym_names += ['resnetv20_conv0_fwd']
+        elif model_name.find('vgg') != -1:
+            excluded_sym_names += ['vgg0_conv0_fwd']
+        elif model_name.find('squeezenet1') != -1:
+            excluded_sym_names += ['squeezenet0_conv0_fwd']
+        elif model_name.find('mobilenet') != -1 and model_name.find('v2') == -1:
+            excluded_sym_names += ['mobilenet0_conv0_fwd']
+        elif model_name.find('mobilenet') != -1 and model_name.find('v2') != -1:
+            excluded_sym_names += ['mobilenetv20_conv0_fwd']
+        elif model_name.find('inceptionv3') != -1:
+            excluded_sym_names += ['inception30_conv0_fwd']
```

Comment:
Why? Is there an accuracy issue without those exclusions?

Reply:
Not for accuracy, but for performance purpose. This could be removed once more bfloat16 hardware added.

Reply:
Please add a comment for this temp performance solution and will convert all conv layers later.